packages/plugins/blocks/blocks-antd/src/blocks/Menu/Menu.js

```diff
@@ -288,16 +267,13 @@ const MenuComp = ({
                                   )
                                 }
                               >
-                                <MenuTitle
-                                  basePath={basePath}
-                                  Link={Link}
-                                  linkStyle={methods.makeCssClass(subLinkGroup.style, true)}
-                                  id={subLinkGroup.id}
-                                  makeCssClass={methods.makeCssClass}
-                                  pageId={subLinkGroup.pageId}
-                                  properties={subLinkGroup.properties}
-                                  url={subLinkGroup.url}
-                                />
+                                <Link
```

Comment:
So will menus now support `urlQuery` and `input`?

Reply:
Amazing right 😱 should I update the docs?